src/overlays/actors/ovl_En_Az/z_en_az.c

```diff
@@ -1680,26 +1756,26 @@ static struct_80A98F94 D_80A9930C[5][4] = {
     { { 0, 1900.0f }, { 4, 1600.0f }, { 10, 2200.0f }, { 16, 1900.0f } },
     { { 0, 1900.0f }, { 7, 1400.0f }, { 16, 1900.0f }, { 0, 0.0f } },
 };
-static Vec3f D_80A993AC[3] = {
+static Vec3f D_80A993AC[] = {
     { 1.0f, 1.0f, 0.0f },
     { 0.9f, 0.9f, 120.0f },
     { 0.95f, 0.95f, 240.0f },
 };
-static Vec3f D_80A993D0[3] = {
+static Vec3f D_80A993D0[] = {
     { 1.5f, 1.5f, 0.0f },
     { 1.2f, 1.2f, 120.0f },
     { 1.35f, 1.35f, 240.0f },
 };
-static TexturePtr D_80A993F4[4] = {
-    object_az_Tex_00F918,
-    object_az_Tex_010118,
-    object_az_Tex_010918,
-    object_az_Tex_011118,
+static TexturePtr D_80A993F4[] = {
+    gBeaverYoungerBrotherEye1Tex,
+    gBeaverYoungerBrotherEye2Tex,
+    gBeaverYoungerBrotherEye3Tex,
+    gBeaverYoungerBrotherEye4Tex,
 };
-static TexturePtr D_80A99404[3] = {
-    object_az_Tex_016018,
-    object_az_Tex_016818,
-    object_az_Tex_017018,
+static TexturePtr D_80A99404[] = {
+    gBeaverYoungerBrotherBelt1Tex,
+    gBeaverYoungerBrotherBelt2Tex,
+    gBeaverYoungerBrotherBelt3Tex,
 };
```

Comment:
Can name these arrays, then. Is there no way to distinguish these textures?

Reply:
For the eyes, almost certainly not. It's just a repeating sequence:
![beaver_younger_brother_eye_1 rgba16](https://user-images.githubusercontent.com/12245827/191655225-e4d80f97-1b32-4651-8d50-65e99a547acb.png)
![beaver_younger_brother_eye_2 rgba16](https://user-images.githubusercontent.com/12245827/191655234-c703278f-1fcd-4a17-bc88-68c34a0f7b22.png)
![beaver_younger_brother_eye_3 rgba16](https://user-images.githubusercontent.com/12245827/191655237-5cf231f3-af26-4615-9cbf-b20664605b75.png)
![beaver_younger_brother_eye_4 rgba16](https://user-images.githubusercontent.com/12245827/191655242-dbc026ff-6f50-4ed6-af7b-4a9bf03cf86e.png)

For the belt, maybe you could name them based on the colors? But it's an infinitely repeating sequence just like the eyes:
![beaver_younger_brother_belt_1 rgba16](https://user-images.githubusercontent.com/12245827/191655420-f3b50270-5503-4277-9029-d99037e2fb2d.png)
![beaver_younger_brother_belt_2 rgba16](https://user-images.githubusercontent.com/12245827/191655426-7781eaac-c0ef-4717-9891-4cdd544a446f.png)
![beaver_younger_brother_belt_3 rgba16](https://user-images.githubusercontent.com/12245827/191655434-d0be8148-1e6b-4f99-ac01-115a430501d4.png)

Reply:
Oh right, it's specifically the propeller thing. Hmm, yes, I'd say red,green,blue for those in that order. Nothing we can do about the eyes, though, except start at 0.

Reply:
I can change the propeller thing, but I don't think it's common for us to zero-index texture names like that.

Reply:
I think some of us discussed it before and it was considered a good idea, but there's plenty of effects that will be done later anyway so I'm not going to worry about it for now.